version.properties

```diff
@@ -1,2 +1,2 @@
-versionName=7.55-rc-1
-versionCode=9181
\ No newline at end of file
+versionName=7.55-rc-2
+versionCode=9184
```

Comment:
My GitHub token expired and had to retrigger the beta build twice because of which version code is incremented from 9181 to 9184. I manually updated versionName to get the correct one on the next build.